### PR TITLE
Podcasts: add 240, add new podcatchers

### PR DESCRIPTION
**Pull request by bitschmidty**

### CODE REVIEW

**Comment by bitschmidty** [comment]
Added a commit to disable to localizations navigation links (similar to topics pages). Can revisit if there is demand for translations related to the podcast.

**Comment by harding** [comment]
Skimmed the transcript and quickly checked the diff.  Didn't see any problems.

File: _posts/en/podcast/2023-03-02-newsletter-recap.md [hunk 1]
```diff
@@ -0,0 +1,412 @@
...
+parties, but consigned together without using native multisig, but still get the
+signoff of multiple people.  The problem of course is that the device that
+combines the information from the multiple shares does learn, at least
+temporarily, the secret, so the idea is that is some sort of hardware device
+that is trusted to only take the inputs from the users and then not be able to
+export the key.
+
+I know that the Fireblocks I think is MPC-based instead of multisig-based, and
+that way they could use essentially the same solutions for Bitcoin and Ethereum;
+but it also scared some of the cryptographers that I was working with, how new
```
Comment (Xekyo) [inline _posts/en/podcast/2023-03-02-newsletter-recap.md:193]:
```suggestion
but it also scared some of the applied cryptographers that I was working with, how new
```

File: _posts/en/podcast/2023-03-02-newsletter-recap.md [hunk 2]
```diff
@@ -0,0 +1,412 @@
...
+
+**Mike Schmidt**: Excellent, thank you for that overview.  So, those were the
+notable things in HWI.
+
+{:#cln}
+_Core Lightning 23.02rc3_
+
+The next notable release candidate is Core Lightning 23.02rc3, which we've had
+in actually for a couple of weeks now.  And so, if you go back I think to our
+Recap for #238, we had a couple of folks from Blockstream in the Core Lightning
```
Comment (Xekyo) [inline _posts/en/podcast/2023-03-02-newsletter-recap.md:224]: I verified that it was #238. Perhaps we should link here to that episode when we get that it out?

File: _posts/en/podcast/2023-03-02-newsletter-recap.md [hunk 3]
```diff
@@ -0,0 +1,412 @@
...
+_lnd v0.16.0-beta.rc1_
+
+And then the last release we had is LND v0.16.0 and this is a beta release
+candidate.  A couple of things that I noticed were interesting here was a few
+P2P updates for LND, including a bug fix that could lead to some channel updates
+being missed; and then also some changes at the P2P level for gossip updates.
+They've also added the decryption for larger Onion failure messages, and we've
+covered that a few times in the newsletter.  The ability to have these larger
+error messages can enable better pathfinding using what we've talked with Joost,
+I believe, about fat errors.  So I thought that was interesting.  There was also
```
Comment (Xekyo) [inline _posts/en/podcast/2023-03-02-newsletter-recap.md:238]: Possibly link to the episode with Joost here. Perhaps linking to the news item meanwhile would be useful? Or do you think the related news item is already sufficiently linked up?

File: _posts/en/podcast/2023-03-02-newsletter-recap.md [hunk 4]
```diff
@@ -0,0 +1,412 @@
...
+internal wallet for Taproot pubkeys in Tapscripts.  And then, there was a bunch
+of other RPC and wallet updates and bug fixes, so jump into the release notes
+there, it's really comprehensive there.
+
+We've just wrapped up the release candidate section, so I thought maybe you'd
+take this first notable PR that we have, and also this is a time to solicit
+questions from anybody listening.  If you have questions on what we've
+discussed, or maybe a general question as well, raise your hand and we'll try to
+get to you when we wrap up the rest of these PRs.
+
```
Comment (Xekyo) [inline _posts/en/podcast/2023-03-02-newsletter-recap.md:249]: This is skipped in the recording, should we delete?

```suggestion
```

File: _posts/en/podcast/2023-03-02-newsletter-recap.md [hunk 5]
```diff
@@ -0,0 +1,412 @@
...
+consider special-casing that, because an invalid opcode would also mean that the
+transaction is invalid.  So, maybe this is when people are building transactions
+in Bitcoin Core that they want to send to a compatible other network, but
+honestly I'm not quite sure.
+
+**Mike Schmidt**: And, does your answer also apply to the case of an output
+whose script exceeds the maximum script size as well; is that a similar answer?
+
+**Mark Erhardt**: Yes, so I guess this is why.  When we write an output, we do
+not execute the output script, right, we only log funds to the output.  So, if
```
Comment (Xekyo) [inline _posts/en/podcast/2023-03-02-newsletter-recap.md:275]:
```suggestion
not execute the output script, right, we only lock funds to the output.  So, if
```

File: _posts/en/podcast/2023-03-02-newsletter-recap.md [hunk 6]
```diff
@@ -0,0 +1,412 @@
...
+_Bitcoin Core #27068_
+
+Next PR here is Bitcoin Core #27068 and this is also related to passphrases.  It
+sounds like previously, if you had a passphrase and you encrypted your wallet
+using a passphrase that had a special character, an ASCII null character, which
+is 0x00, the passphrase would actually be truncated at the point that there was
+a null character.  So, you may have thought that you put in a 20-character
+passphrase but if you had an ASCII null character at spot 5, let's say, then you
+would have actually those additional characters would be truncated and you would
+have much less passphrase that you thought, and obviously there's some usability
```
Comment (Xekyo) [inline _posts/en/podcast/2023-03-02-newsletter-recap.md:324]:
```suggestion
have a much less secure passphrase that you thought, and obviously there's some usability
```

File: _posts/en/podcast/2023-03-02-newsletter-recap.md [hunk 7]
```diff
@@ -0,0 +1,412 @@
...
+can start experimenting with offers in LDK now.  Murch, any thoughts on LDK and
+offers?
+
+**Mark Erhardt**: Yeah, I've been seeing a bunch of PRs from LDK on offers.  It
+looks like they're moving forward pretty quickly with getting support rolling.
+And I was recently asked by someone how long it will take until offers will be
+widely deployed on the Lightning Network.  So, I think work started about three
+years ago.  The first experimental support was two years ago in, I think,
+C-Lightning, and we have also seen now, in an announcement by LND, that was
+focusing on other things, not working on BOLT12 yet, that they intend to have
```
Comment (Xekyo) [inline _posts/en/podcast/2023-03-02-newsletter-recap.md:388]:
```suggestion
previously focusing on other things, not working on BOLT12 yet, that they intend to have
```

**Comment by Xekyo** [comment]
I listened to the Spaces a bit reading the transcript in parallel, then realized that my podcast catcher already had the actual episode. Switching over, my experience was hugely improved! :grin:

I noticed a few discrepancies, and slight misunderstandings of the transcriptionist. A couple of these actually change the meaning, the rest are just nits.

File: _posts/en/podcast/2023-03-02-newsletter-recap.md (hunk 3 above)
Comment (bitschmidty) [reply _posts/en/podcast/2023-03-02-newsletter-recap.md:238]: Might be enough with the related new item info

File: _posts/en/podcast/2023-03-02-newsletter-recap.md (hunk 4 above)
Comment (bitschmidty) [reply _posts/en/podcast/2023-03-02-newsletter-recap.md:249]: We actually got a second version of the audio after the transcription, which explains this. Will remove.

**Review by Xekyo — APPROVED** [review]
ACK

**Comment by bitschmidty** [comment]
Took transcription suggestions from @Xekyo review, thanks! Squashed. 🚀

**Pull request merged**